caom2-search-server/src/main/resources/META-INF/resources/js/cadc.search.app.js

```diff
@@ -1249,9 +1249,12 @@
 
       var serializedUrl = serializer.getResultStateUrl()
 
+      // Build an href and place it in the bookmark div.
+      var bookmarkLink = '<a href="' + serializedUrl + '" target=_blank>' + serializedUrl + '</a>'
```

Comment:
Nit picky.  For consistency, add double-quotes around the `_blank`.

Reply:
Not a problem! done.